src/learn_to_pick/byom/pytorch_feature_embedder.py

```diff
@@ -0,0 +1,90 @@
+import learn_to_pick as rl_chain
+from sentence_transformers import SentenceTransformer
+import torch
+
+
+class PyTorchFeatureEmbedder:
+    def __init__(self, auto_embed, model=None, *args, **kwargs):
```

Comment:
thinking: what happens in PyTorch if `auto_embed` is off? it will probably fail so maybe we should have it always on?

it doesn't match the other feature embedder constructor, but they are dissimilar in other ways too